app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+
+const app = express();
+
+// custom error <- DELETE this comment later on
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send(err);
+  } else {
+    next();
+  }
+});
```

Comment:
We aren't working on error-handling yet - please remove for now

Reply:
Sorry thats me just defaulting with a set up and a custom error